src/context/Authentication/AuthProvider.js

```diff
@@ -1,16 +1,17 @@
-import AuthContext from './authContext';
-import getToken from '../../services/getToken';
-import AuthReducer from './authReducer';
-import { useReducer, useEffect } from 'react';
-import AsyncStorage from '@react-native-async-storage/async-storage';
-import getUser from '../../services/getUser';
-import setUser from '../../services/setUser';
+import AuthContext from "./authContext";
+import getToken from "../../services/getToken";
+import AuthReducer from "./authReducer";
+import { useEffect, useReducer } from "react";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import getUser from "../../services/getUser";
+import setUser from "../../services/setUser";
+import storage from "../../storage";
 
-const AuthState = ({ children }) => {
+export const AuthProvider = ({ children }) => {
```

Comment:
a little bit not happy with the name AuthProvider even though it makes sense😁. Basically, the AuthState gives us all the current states and the authReducer decides what can happen to your state based on the action, that's why I named it AuthState.

Reply:
Sure. If you aren't happy with it, we can revert it.

Reply:
No, it's okay.